dbms/src/Flash/Executor/ExecutionResult.h

```diff
@@ -0,0 +1,43 @@
+// Copyright 2023 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <Common/Exception.h>
+#include <common/types.h>
+
+namespace DB
+{
+struct ExecutionResult
+{
+    bool is_success;
```

Comment:
```suggestion
    bool success;
```
I think `success` is consistent with the coding style of TiFlash

Reply:
Sadly, this has the same name as function `static ExecutionResult success()`, compile error

Reply:
`successed`

Reply:
I don't think `is_success` is a bad name compared to `successed`.